tikv-control.md

```diff
@@ -230,18 +230,28 @@ The properties can be used to check whether the Region is healthy or not. If not
 
 ### Compact data of each TiKV manually
 
-Use the `compact` command to manually compact data of each TiKV. If you specify the `--from` and `--to` options, then their flags are also in the form of escaped raw key.
+Use the `compact` command to manually compact data of each TiKV.
 
-- Use the `--host` option to specify the TiKV that needs to perform compaction.
-- Use the `-d` option to specify the RocksDB that performs compaction. The optional values are `kv` and `raft`.
+- Use the `--from` and `--to` options to specify the compact range in the form of escaped raw key. If not set, the whole range will be compacted.
+- Use the `--region` option to compact range of a specific region. If set, `--from` and `--to` will be ignored.
+- Use the `--db` option to specify the RocksDB that performs compaction. The optional values are `kv` and `raft`.
 - Use the `--threads` option allows you to specify the concurrency for the TiKV compaction and its default value is `8`. Generally, a higher concurrency comes with a faster compaction speed, which might yet affect the service. You need to choose an appropriate concurrency count based on your scenario.
```

Comment:
[This line](https://github.com/pingcap/docs-cn/pull/9886/files#diff-ff89854f38035fc717cda9246e345cffc32f58be987165f3529f06b3a4586065R289) is missing in the translation. Please add it here.

Reply:
The `--data-dir` option is removed in https://github.com/pingcap/docs-cn/pull/11026